tests/test_ote_api.py

```diff
@@ -12,42 +12,43 @@
 # See the License for the specific language governing permissions
 # and limitations under the License.
 
-import glob
-import io
-import os.path as osp
-import random
 import time
 import unittest
-import warnings
-from concurrent.futures import ThreadPoolExecutor
-from subprocess import run  # nosec
-from typing import Optional
 
+import glob
+import io
 import numpy as np
+import os.path as osp
+import random
 import torch
+import warnings
 from bson import ObjectId
-from ote_sdk.test_suite.e2e_test_system import e2e_pytest_api
+from concurrent.futures import ThreadPoolExecutor
```

Comment:
What's the ordering rule here?

Reply:
Yes, it seems very random, that was not intended. Fixed